artifacts

```diff
@@ -24,7 +24,19 @@ setup() {
 }
 
 prolong() {
-    echo "TODO"
+    if [ -z "${NAME}" ]; then
+      echo ${NAME}
+      exit 2
+    fi
```

Comment:
You have set the default value of name as `default` on the `action.yaml` so I think we will never hit this condition.